docs/build.md

```diff
@@ -0,0 +1,144 @@
+# vue build
+
+`vue build` command gives you a zero-configuration development setup, install once and build everywhere.
+
+## Features
+
+- **Not a boilerplate**: run a single command to develop your app
+- **Out of the box**: ES2015, single-file component with hot reloading and custom CSS preprocessors
+- **Customizable**: populate a `~/.vue/webpack.config.js` for custom webpack config
+- **Single-file component mode**: just run `vue build Component.vue` and boom!
+
+## Get started
+
+Populate a component `./component.vue` in your project:
+
+```vue
+<template>
+  <h2>Hello World!</h2>
+</template>
+```
+
+And then just run `vue build component.vue` and go to `http://localhost:4000`
+
+To build for production:
```

Comment:
both single-file component and regular entry are ok, in fact I feel directly test component doesn't make much sense, because component always needs props 😂

The `build for production` means built as a complete web app in production, maybe add a `distribute component` section to clarify more?